frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -1067,13 +1074,32 @@ class TableInteractive extends Component {
 
     const headerHeight = this.props.tableHeaderHeight || HEADER_HEIGHT;
     const gutterColumn = this.state.showDetailShortcut ? 1 : 0;
-    const shortcutColumn = Number(!question?.isArchived());
-    const query = question?.query();
-    const info = query && Lib.queryDisplayInfo(query);
+
+    const shortcutActions =
+      question && mode?.clickActions
+        ? mode.clickActions.flatMap(action =>
```

Comment:
`question && mode?.clickActions?.flatMap`

Reply:
and do we really need to iterate over all click actions just to understand if it's greater than 0?